tests/Integration/Service.Tests/TariffModuleSpecs/WhenAddingTariff.cs

```diff
@@ -26,10 +26,10 @@ public void SetUp()
                 Description = "test-case"
             };
             var tariff = new Tariff
-                             {
-                                 TariffCode = "test",
-                                 Description = "test-case"
-                             };
+            {
+                TariffCode = "test",
+                Description = "test-case"
```

Comment:
I prefer the previous indentation personally

Reply:
👍